test/e2e/framework/util.go

```diff
@@ -4442,19 +4452,44 @@ const (
 	IPv6PingCommand PingCommand = "ping6"
 )
 
+// NcCommand is the type to hold nc command
+type NcCommand string
+
+const (
+	// IPv4NcCommand is a nc command for IPv4
+	IPv4NcCommand NcCommand = "nc"
```

Comment:
the nc version included in `agnhost` support both IPv4 and IPv6

Reply:
Got it. Will use ```nc``` from ```agnhost```.